### PR TITLE
Use PHP-CS-Fixer as a formatter

### PR DESCRIPTION
**Pull request by mniewrzal**

### CODE REVIEW

File: fixtures/format.php [hunk 1]
```diff
@@ -9,12 +9,16 @@ class TestClass
...
         $testVariable = 123;
-        
-                if (  empty($testParameter)){
-                    echo 'Empty';
+
+        if (  empty($testParameter))    {
+            echo 'Empty';
         }
     }
 }
```
Comment (felixfbecker) [inline fixtures/format.php]: Why the changes in this file?

Comment (mniewrzal) [inline fixtures/format.php]: CS-Fixer works in different way than Pretty Printer. PP is converting ast into source code according to some rules. CS-Fixer search for invalid part of code and fix just that part. Looks that not all parts of source code are covered by cs-fixer and for example "namespace       TestNamespace;" wont be converted to "namespace TestNamespace;". I still need to check if its by design or its a bug.

**Comment by mniewrzal** [comment]
Code Sniffer formatter looks to be better choice in my opinion.
